### ISSUE TITLE
Separator is not found, and chunk exceed the limit

### ISSUE THREAD

**Issue by xabil** [just-work/fffw#74]
I am trying to transcode using `django-video-transcoding` which uses fffw library, and for a few streams I am getting `Separator is not found, and chunk exceed the limit`. I tried running ffmpeg with the exact command which`django-video-transcoding` logs and it works fine.

**Comment by tumb1er** [comment]
Could you please provide full stack trace?

**Comment by tumb1er** [comment]
It will be useful to have full ffmpeg command line also.

**Comment by xabil** [comment]
I am not getting a stack trace, as the ffmpeg is running in a different context in `django-video-transcoding`. Here is the ffmpeg command:
```
ffmpeg -loglevel repeat+level+info -y -i https://bucket-name.s3.amazonaws.com/geri.mp4 -filter_complex "[0:v]scale=w=1920:h=1080[vout0]" -map "[vout0]" -c:v libx264 -force_key_frames "expr:if(isnan(prev_forced_t),1,gte(t,prev_forced_t+4))" -crf 23 -preset slow -maxrate 5000000 -bufsize 10000000 -profile:v high -g 47 -r 23.976 -map 0:a -c:a aac -b:a 192000 -ar 48000 -ac 2 -f mp4 /tmp/video-1-jcc2bgcu/60f9f76907fb494eab20c71f05e4de0f1080p.mp4
```
I did some investigations and the error seems to be coming from asyncio. Maybe providing a larger buffer works, or maybe make it configurable?

**Comment by tumb1er** [comment]
Looks like this the same error that we are trying to fix in #75. It happens because asyncio StreamReader doesn't respect CR as a line separator and it's not configurable. 
FFMPEG uses CR to output transcoding progress and there may be no newlines at all till the end of processing.

![image](https://user-images.githubusercontent.com/1587241/97783364-32764700-1ba8-11eb-91ce-d71610def7a5.png)

Could you please install fffw from https://github.com/tumb1er/fffw/tree/universal_line_reader and check whether the error is gone?

**Comment by xabil** [comment]
I'll give it a try and let you know. Thanks for quick response.

**Comment by tumb1er** [comment]
@xabil just released 3.1.1 with a fix for related issue. Please fill free to repoen this issue if your problem persists.

**Comment by xabil** [comment]
It is fixed with the latest release 3.1.1, thanks tumb1er.
